server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json

```diff
@@ -297,118 +297,82 @@
               {
                 "tiles": [
                   {
-                    "rankingTileSpec": {
-                      "rankingCount": 10,
-                      "showHighest": true
-                    },
                     "statVarKey": [
-                      "sdg/NY_GDP_PCAP"
+                      "sdg/SI_POV_EMP1.AGE--Y15T24",
+                      "sdg/SI_POV_EMP1.AGE--Y15T24__SEX--F",
+                      "sdg/SI_POV_EMP1.AGE--Y15T24__SEX--M"
                     ],
-                    "title": "Annual Growth Rate of Real GDP Per Capita in Administrative Area 1 Places of India (${date})",
-                    "type": "RANKING"
-                  }
-                ]
-              },
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "sdg/NY_GDP_PCAP"
-                    ],
-                    "title": "Annual Growth Rate of Real GDP Per Capita in Administrative Area 1 Places of India (${date})",
-                    "type": "MAP"
+                    "title": "Employed Population 15 to 24 Years Old Below International Poverty Line, by Sex (%) in India",
```

Comment:
It seems this is better than LHS for poverty query, so find losing sdg/NY_GDP_PCAP